Dockerfile

```diff
@@ -25,6 +25,12 @@ COPY src/scripts/* /operator/
 COPY operator/target/weblogic-kubernetes-operator-$VERSION.jar /operator/weblogic-kubernetes-operator.jar
 COPY operator/target/lib/*.jar /operator/lib/
 
+# make the operator run with a non-root user id (1000 is the `oracle` user)
+RUN groupadd -g 1000 oracle
+RUN useradd -d /operator -M -s /bin/bash -g 1000 -u 1000 oracle 
+RUN chown -R 1000:1000 /operator
```

Comment:
Can you please check what this does to the image size?  It will likely be better to create the user/group first and then add the chown statement to the COPY: https://docs.docker.com/engine/reference/builder/#copy

Reply:
As is, the difference is about +56MB.  I can try to optimize it if you want.

[root@mark weblogic-kubernetes-operator]# docker images | grep operator
weblogic-kubernetes-operator                               2.3.0               188c2b0232b8        7 seconds ago       532MB
weblogic-kubernetes-operator-79168                         latest              83033cd3ddbb        About an hour ago   586MB

Reply:
Yes, please.  At least move the user/group creation up and then add the chown to the existing COPY statements.

Reply:
ok i made a change and I got it to zero size change. will push that new commit now.

[root@mark weblogic-kubernetes-operator]# docker images | grep operator
weblogic-kubernetes-operator-79168                         latest              3c81210ab060        56 seconds ago      532MB
weblogic-kubernetes-operator                               2.3.0               188c2b0232b8        13 minutes ago      532MB

Reply:
new test run: http://build.weblogick8s.org:8080/job/weblogic-kubernetes-operator-quicktest/883/console

Reply:
What does this do to the current approach of using the initContainer to change the ownership on dynamically allocated PVs?

Reply:
This is a different use case.  This is for the operator, itself, not for the generated domain pods.

Reply:
new run (855) has the test passing: 
01-22-2020 19:14:40 INFO  
                                    oracle.kubernetes.operator.ItJrfPvWlst testJrfDomainOnPvUsingWlst SUCCESS - testJrfDomainOnPvUsingWlst